### ISSUE TITLE
Scene rendering stops upon speech completion

### ISSUE THREAD

**Issue by M-Anne-M** [aws-samples/amazon-sumerian-hosts#30]
Once any host completes it's speech the entire Babylon scene just quits rendering and hold a still image on the screen.  It throws this error and then points to a variety of places in the different js files for the speech components:
Speech.js?2a41:128 Uncaught TypeError: Cannot set property currentTime of #<e> which has only a getter

It does however keep playing speech when you click the start button, only the visuals are stopped from working.  It started doing that around three weeks ago on my computer out of the blue.  Essentially without changing a single thing in the code it just quit working overnight.  Using an old version of the repo (the one where the poi feature was just released) does still work, but the newest version of it does not.   I tried completely re-downloading and installing the repo several times, and get the same error each time.  Any help is appreciated, thanks!

**Comment by jkeys-ecg-nmsu** [comment]
@c-morten I can provide the actual logs over email if they would help.

Stack trace: 
![image](https://user-images.githubusercontent.com/22964702/97895336-e551bc80-1cf0-11eb-9763-f1c684df2a9e.png)

Specifically, it errors on this line in Speech.js: 

![image](https://user-images.githubusercontent.com/22964702/97895425-fef30400-1cf0-11eb-9abc-477e9bef1658.png)

**Comment by jkeys-ecg-nmsu** [comment]
Also, this bug was replicated on 1.3.2 of this repo, and babylonjs@4.2.0-beta.21.

When I downgraded to babylonjs@4.2.0-beta.9, the error went away.

**Comment by c-morten** [comment]
^ Thanks @jkeys-ecg-nmsu, this is really helpful info. I'll take a look at Babylon's latest preview build later this evening and see what has changed in the audio system.

**Comment by KISSMonX** [comment]
> Also, this bug was replicated on 1.3.2 of this repo, and babylonjs@4.2.0-beta.21.
> 
> When I downgraded to babylonjs@4.2.0-beta.9, the error went away.

This works! thank you very much.
Look forward to further updates.

**Comment by KISSMonX** [comment]
> ^ Thanks @jkeys-ecg-nmsu, this is really helpful info. I'll take a look at Babylon's latest preview build later this evening and see what has changed in the audio system.

Looks like BabylonJS preview version has compatibility issues with Sumerian.  Hope the new preview version of BabylonJS will fix this.

**Comment by DC2009** [comment]
I temporary solved the problem by bypassing the library Speech stop. 
I think it's enough to update the babylon function by removing the line highlighted here https://github.com/aws-samples/amazon-sumerian-hosts/issues/30#issuecomment-720593374
```
import HOST from "amazon-sumerian-hosts/dist/host.babylon";

/**
 * @extends babylon/Speech
 */
class Speech extends HOST.aws.Speech {
  /**
   * @constructor
   *
   * @param {Babylon.js/TextToSpeechFeature} speaker - The owner of the Speech
   * that will emit speechmark messages.
   * @param {string} text - The text of the speech.
   * @param {Array.<Object>} [speechmarks=[]] - An array of speechmark objects representing
   * the text and timing of the speech.
   * @param {Object} audioConfig - Object containing audio and url.
   * @param {external:Audio} audioConfig.audio - Playable audio object.
   */

  stop() {
    this._audio.stop();
    this._pauseAudio();

    if (this._promise) {
      this._promise.resolve();
      this._promise = null;
    }

    this._playing = false;
  }
}

export default Speech;
```

**Comment by c-morten** [comment]
Fixed in v1.3.3
